### PR TITLE
New trainee branch

### PR DESCRIPTION
**Pull request by TraineeMax**

### CODE REVIEW

**Comment by MaxLaurel** [comment]
it is ok

File: ToDoTasks/Controllers/LoginViewController.swift [hunk 1]
```diff
@@ -9,15 +9,15 @@ import UIKit
 import Firebase
 
 class LoginViewController: UIViewController, UITextFieldDelegate {
-
+///Hey its a traineeCommit
     private var mainLabel: UILabel = {
-        var mainLabel = UILabel()
```
Comment (MaxLaurel) [inline ToDoTasks/Controllers/LoginViewController.swift:14]: why did you delete this?

Comment (TraineeMax) [reply ToDoTasks/Controllers/LoginViewController.swift:14]: I dunno

**Review by MaxLaurel — APPROVED** [review]
ok you can merge this
